addons/controllers/antreaconfig_controller_test.go

```diff
@@ -227,4 +254,212 @@ var _ = Describe("AntreaConfig Reconciler and Webhooks", func() {
 			Expect(k8sClient.Update(ctx, config)).ToNot(Succeed())
 		})
 	})
+
+	When("antrea config is precreated with {CLUSTER_NAME}-{package-short-name}-package", func() {
+
+		BeforeEach(func() {
+			clusterName = "antrea-custom-cb-cluster"
+			clusterNamespace = "antrea-custom-cb-ns"
+			configName = util.GeneratePackageSecretName(clusterName, constants.AntreaDefaultRefName)
+			tkrString = "v1.23.1"
+			clusterResourceFilePath = "testdata/antrea-custom-tkg-system.yaml"
+			clusterInfraName = "custom-cb-docker-cluster"
+			newDefaultMTU = "8901"
+			ns := &v1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: clusterNamespace,
+				},
+			}
+			err := k8sClient.Create(ctx, ns)
+			if err != nil {
+				Expect(apierrors.IsAlreadyExists(err)).To(BeTrue())
+			}
+
+		})
+
+		It("should add cluster as owner reference to antrea config", func() {
+			// define custom resources in cluster namespace
+			f, err := os.Open("testdata/antrea-custom-cb-ns.yaml")
```

Comment:
This file could be split into two, one with the current resources and the other file could be the section sited in the readme.md file. Thus we can test the example and make sure we don't break it